components/InputTypeLocation.js

```diff
@@ -65,78 +137,6 @@ class InputTypeLocation extends React.Component {
 
     return (
       <div className={classNames('InputTypeLocation', this.props.className)}>
```

Comment:
I don't think we need this anymore

```suggestion
      <div>
```